common/Views/DevHomePage.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevHome.Common.Views;
+
+public class DevHomePage : Page
+{
+    public DevHomePage()
+    {
+        Loaded += (s, e) =>
```

Comment:
The majority of pages and user controls work well with this event firing first.

I did use the autofocus behavior offered by the community toolkit on one page that needed a tail-end focus.

This will not be changed in this PR.  If we can figure out a way to reference a callback and _not_ pass it down the call stack, I'm all ears.

I don't believe a behavior will work in this scenario because
1. No view elements to attach to.
2. No global "Please set focus" flag.

In regards to "2" I could make a bool and ask each page "Please set this to true when you want to focus" but this means the code-behind needs to pass the bool into the view model.